### PR TITLE
feat: add Scaleway template

### PR DESCRIPTION
**Pull request by dirien**
Hi everyone,

This PR adds templates for the pulumiverse Scaleway provider

@stack72, let me know if there is something missing. I will correct it.

### CODE REVIEW

**Review by ringods — CHANGES_REQUESTED** [review]
@jkodroff all the package dependencies have changed. @jaxxstorm moved this package to his own organization. For the correct values, see here:

https://www.pulumi.com/registry/packages/scaleway/

**Comment by dirien** [comment]
HI @ringods,

updated the locations. Should be fine now to merge? WDT?

File: scaleway-javascript/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+"use strict";
+const pulumi = require("@pulumi/pulumi");
+const scaleway = require("@pulumiverse/scaleway");
```
Comment (ringods) [inline scaleway-javascript/index.js:3]: Package should be `@lbrlabs/pulumi-scaleway`

https://www.npmjs.com/package/@lbrlabs/pulumi-scaleway

File: scaleway-python/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+pulumi==3.38.0
+brlabs_scaleway==1.3.0
```
Comment (ringods) [inline scaleway-python/requirements.txt:2]: Package name is not correct: should be `lbrlabs-scaleway`

https://pypi.org/project/lbrlabs-scaleway/

File: scaleway-python/__main__.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import pulumi
+import pulumiverse_scaleway as scaleway
```
Comment (ringods) [inline scaleway-python/__main__.py:2]: Import is wrong, should be `lbrlabs_scaleway`

https://pypi.org/project/lbrlabs-scaleway/

File: scaleway-typescript/index.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as scaleway from "@pulumiverse/scaleway";
```
Comment (ringods) [inline scaleway-typescript/index.ts:2]: Same here.

**Review by ringods — CHANGES_REQUESTED** [review]

File: scaleway-typescript/index.ts (hunk 4 above)
Comment (dirien) [reply scaleway-typescript/index.ts]: aaaaaah, sorry! Done!

File: scaleway-python/__main__.py (hunk 3 above)
Comment (dirien) [reply scaleway-python/__main__.py]: aaaaaah, sorry! Done!

File: scaleway-python/requirements.txt (hunk 2 above)
Comment (dirien) [reply scaleway-python/requirements.txt]: aaaaaah, sorry! Done!

**Review by ringods — APPROVED** [review]

**Review by ringods — APPROVED** [review]

**Comment by dirien** [comment]
Hi @cnunciato, can you have a look into this PR? Many thanks!

File: oci-typescript/package.json
Comment (cnunciato) [inline oci-typescript/package.json:1]: Could we remove this change from the PR? It doesn't appear to be related to the others.

Comment (dirien) [reply oci-typescript/package.json:1]: ups.. Done, removed it! 😅

**Comment by cnunciato** [comment]
Chatted with @dirien this morning and offered a few suggestions ([along with a PR](https://github.com/dirien/templates/pull/1/files)) to bring these into closer alignment with our starter templates.

**Review by cnunciato — APPROVED** [review]
🤜 🤛
